compiler/bcq-tools/README.md

```diff
@@ -76,3 +76,38 @@ const(value=[1, 2, 3, -3], name='const3')
 ### Caution
 
 - Generated output_arrays will be start with comma.
+
+## generate_bcq_metadata
+
+### Purpose
+
+To apply BCQ, additional information which is not included in original model is needed.
```

Comment:
Somewhat vague sentence.

Reply:
Which point is vague?
If `additional information` is vague, next sentence is saying with an example. However if other thing is vague, could you let me know? :)

Reply:
Yes, `additional information` is somewhat vague.